server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/service/navigator/impl/WebServiceNavigator.java

```diff
@@ -111,16 +107,19 @@ public List<WebNavigatorNodeInfo> getNavigatorNodeChildren(
                     // Skip empty folders. Folder may become empty if their nested elements are provided by UI plugins.
                     continue;
                 }
-                if (!CommonUtils.toBoolean(onlyFolders) || node instanceof DBNContainer) {
+                if (CommonUtils.toBoolean(onlyFolders)) {
```

Comment:
Did you forget the `!` while refactoring?
```suggestion
                if (!CommonUtils.toBoolean(onlyFolders)) {
```